### ISSUE TITLE
Xbox Game Prediction

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#370]
Hey LGM-SoC participants, good to see you.

This project will help us in predicting which Xbox game a visitor will be most interested in based on their search query using the dataset given

Dataset : https://www.kaggle.com/c/acm-sf-chapter-hackathon-small/data

If you like to work on this issue, follow the given guidelines for code as well as for the README. You can try to compare it using different algorithms.

**Comment by aswathcm29** [comment]
I'd like to contribute to this issue and the repository. I'm excited to work on predicting Xbox game interests based on search queries and explore different algorithms for this task. 

It'd be great if you assign it to me

**Comment by prathimacode-hub** [comment]
Sure, issue is assigned to you. 
@aswathcm29

**Comment by aswathcm29** [comment]
Yup ! Thanks for assigning it to me @prathimacode-hub ! ....I'll complete it asap and raise the PR

**Comment by prathimacode-hub** [comment]
Any update? @aswathcm29

**Comment by aswathcm29** [comment]
> Any update? @aswathcm29

Yes @prathimacode-hub im working on it will make it done within tmrw evening...
